src/main/webapp/app/entities/responsibility/responsibility.service.ts

```diff
@@ -35,7 +35,7 @@ export class ResponsibilityService {
 
   findAllByProject(project: IProject, req?: any): Observable<EntityArrayResponseType> {
     const options = createRequestOption(req);
-    return this.http.get<IResponsibility[]>(`${this.resourceUrl}/${project.id!}/responsibilities`, {
+    return this.http.get<IResponsibility[]>(`${this.resourceUrl}/project/${project.id!}`, {
```

Comment:
the reason it broke is actually because I moved it from `project.service.ts` into this one in PR #62. which changes the `resourceUrl`. sad.